Changes

```diff
@@ -2,6 +2,16 @@ Revision history for Perl extension Test::PostgreSQL.
 
 {{$NEXT}}
 
+1.27_02
+    - Use pg_version attribute to better handle -b and -f psql switches
+      depending on installed PostgreSQL version
+    - Updated docs to reflect the changes
+    - Tests are passing with PostgreSQL 9.3+
+
+1.27_01
+    - pg_version attribute holds PostgreSQL version detected at startup
+    - Improved test diagnostics
+
```

Comment:
If this is going into a new release, should the changes be under {{NEXT}} rather than the dev-release versions?

Reply:
Right, these were probably remains of trial releases. I'll make the change.